dbclient/common/src/main/java/io/helidon/dbclient/common/AbstractStatement.java

```diff
@@ -37,6 +38,10 @@
  * @param <R> the result type of the statement as returned by {@link #execute()}
  */
 public abstract class AbstractStatement<S extends DbStatement<S, R>, R> implements DbStatement<S, R> {
+
+    /** Local logger instance. */
+    private static final Logger LOG = Logger.getLogger(AbstractStatement.class.getName());
```

Comment:
1. Use the name `LOGGER` to be consistent
2. The instance is never used, so why is it added?
3. Only changes in formatting in this class

Reply:
Fixed.